docs/data/sql_connectors.yml

```diff
@@ -169,22 +169,26 @@ upsert-kafka:
 kinesis:
     name: Amazon Kinesis Data Streams
     category: connector
-    maven: flink-connector-kinesis
-    sql_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-kinesis/$version/flink-sql-connector-kinesis-$version.jar
+    versions:
+        - versions: universal
+          maven: flink-connector-kinesis
+          sql_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-kinesis/4.0.0-1.17/flink-sql-connector-kinesis-4.0.0-1.17.jar
```

Comment:
hm... should `1.17` be `$version` instead?

Reply:
No the `$version` variable is the full version, example, `1.17.0`. We need the short version. There is background work on improving the shortcodes to deal with external connectors more graefully